coreprofile/11/_index.md

```diff
@@ -21,15 +25,15 @@ The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifi
 
 * [Jakarta EE Core Profile 11 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/core-profile-11)
     * [Jakarta EE Platform 11 Release Plan](https://jakartaee.github.io/platform/jakartaee11/JakartaEE11ReleasePlan)
-* [Jakarta EE 11 M4 core profile Specification Document](./jakarta-coreprofile-spec-11.0-M4.pdf) (PDF)
-* [Jakarta EE 11 M4 core profile Specification Document](./jakarta-coreprofile-spec-11.0-M4.html) (HTML)
-* [Jakarta Core Profile 11 M4 Javadoc](./apidocs)
+* [Jakarta EE 11 core profile Specification Document](./jakarta-coreprofile-spec-11.0.pdf) (PDF)
+* [Jakarta EE 11 core profile Specification Document](./jakarta-coreprofile-spec-11.0.html) (HTML)
+* [Jakarta Core Profile 11 Javadoc](./apidocs)
 * Maven coordinates
-   * [jakarta.platform:jakarta.jakartaee-core-api:jar:11.0.0-M4](https://central.sonatype.com/artifact/jakarta.platform/jakarta.jakartaee-core-api/11.0.0-M4/jar)
+   * [jakarta.platform:jakarta.jakartaee-core-api:jar:11.0.0](https://central.sonatype.com/artifact/jakarta.platform/jakarta.jakartaee-core-api/11.0.0/jar)
 
 <!--
```

Comment:
This needs to be uncommented out to make the TCK visible.

Reply:
Fixed in last push